docs/marine-book/marine-rust-sdk/developing/call-parameters.md

```diff
@@ -4,24 +4,62 @@ There is a special API function `marine_rs_sdk::get_call_parameters()` that retu
 
 ```rust
 pub struct CallParameters {
-    /// Peer id of the AIR script initiator.
-    pub init_peer_id: String,
+    /// Parameters of the particle that caused this call.
+    pub particle: ParticleParameters,
 
     /// Id of the current service.
     pub service_id: String,
 
     /// Id of the service creator.
     pub service_creator_peer_id: String,
 
-    /// Id of the host which run this service.
+    /// PeerId of the peer who hosts worker with this service.
```

Comment:
somewhere along the way, marine book needs to reference/define the term worker, e.g., https://github.com/fluencelabs/dev-rewards/issues/46